modules/fb_instant_articles_display/src/DrupalInstantArticleDisplay.php

```diff
@@ -550,4 +553,71 @@ private function fieldFormatSocialElement($items, $body) {
       $body->addChild($social);
     }
   }
+
+  /**
+   * Add ads if configured in settings
+   */
+  public function addAdsFromSettings() {
```

Comment:
@simonengelhardt would this make more sense in the base class? This isn't really specific to a Drupal fieldable entity (which to me is the scope that the Display submodule adds).